src/label/Label.ts

```diff
@@ -45,12 +44,6 @@ export const LabelBase = ThemeableMixin(WidgetBase);
 
 @theme(baseCss)
 export default class Label extends LabelBase<LabelProperties>  {
-	@onPropertiesChanged()
-	protected onPropertiesChanged(event: PropertiesChangeEvent<this, LabelProperties>) {
```

Comment:
Ah, ok. I thought the registry mixin mandated that a `registry` is always passed.